python/ray/rllib/ddpg/apex.py

```diff
@@ -4,28 +4,50 @@
 
 from ray.rllib.ddpg.ddpg import DDPGAgent, DEFAULT_CONFIG as DDPG_CONFIG
 
-APEX_DDPG_DEFAULT_CONFIG = dict(DDPG_CONFIG,
-                                **dict(
-                                    optimizer_class="ApexOptimizer",
-                                    optimizer_config=dict(
-                                        DDPG_CONFIG["optimizer_config"],
-                                        **dict(
-                                            max_weight_sync_delay=400,
-                                            num_replay_buffer_shards=4,
-                                            debug=False,
-                                        )),
-                                    n_step=3,
-                                    num_workers=32,
-                                    buffer_size=2000000,
-                                    learning_starts=50000,
-                                    train_batch_size=512,
-                                    sample_batch_size=50,
-                                    max_weight_sync_delay=400,
-                                    target_network_update_freq=500000,
-                                    timesteps_per_iteration=25000,
-                                    per_worker_exploration=True,
-                                    worker_side_prioritization=True,
-                                ))
+
+def merge_dicts(d1, d2):
+    """Merge two dicts and return a new dict that's their union."""
+    d = d1.copy()
+    d.update(d2)
+    return d
+
+
+APEX_DDPG_DEFAULT_CONFIG = merge_dicts(
```

Comment:
I think we don't need `merge_dicts`. 

You can probably just do 
```
APEX_DDPG_DEFAULT_CONFIG = DDPG_CONFIG.copy()
APEX_DDPG_DEFAULT_CONFIG.update({...})
```
Also, the formatting for the dict is not really nice - let's make sure keys and values are on the same line.

Reply:
I thought about the .copy and .update but it made updating the inner dict (optimizer config) awkward. In python 3.5, you can just do `new_dict = {**old1,**old2}`, but this doesn't work on 2.7.

Reply:
Oh, I see. In that case let's keep as is and then make `_merge_dicts` either a utility file or a helper function.